cmd/plasmacli/subcmd/query/root.go

```diff
@@ -21,3 +26,51 @@ var queryCmd = &cobra.Command{
 	Use:   "query",
 	Short: "Query information related to the sidechain",
 }
+
```

Comment:
TxInfo should go here as well. It would be better to consolidate it into 1 file. Maybe a `query.go` file instead of in `root.go`